### PR TITLE
✨#501 [Feature]: Add umami analytics for reactplayIo 

### LINKED ISSUE #501: ✨ [Feature request]: Add umami analytics for reactplayIo
Open-source Umami is an alternative to Google Analytics that focuses on privacy. Umami provides you with a powerful web analytics solution that respects your users' privacy. Furthermore, you have complete control over your data when you self-host [Umami](https://umami.is/docs/about).

https://blog.reactplay.io/how-to-self-host-your-own-website-analytics-with-umami

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

I worked on adding support for analytics for the reactplayio website.

Fixes #501

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -34,6 +34,12 @@
     <title>
       ReactPlay - One app to learn, create, and share ReactJS projects.
     </title>
+    <script
+      async
+      defer
+      data-website-id="e9be9998-e02c-48ee-b0e4-8456ca5c57a1"
```
Comment (atapas) [inline public/index.html:40]: What is this id related to? Is it any sensitive info to come from the env file?

File: src/common/home/Home.jsx [hunk 2]
```diff
@@ -94,9 +92,12 @@ const Home = () => {
                 </div>{" "}
               </span>
             </a>
-          </div>          
+          </div>
           <div className="video">
-            <YoutubeVideoEmbed videoId="XrWonrp296w" title="🤩 Introducing ReactPlay - An Opensource Platform to Create, Learn, and Share ReactJS Projects" />
+            <YoutubeVideoEmbed
```
Comment (atapas) [inline src/common/home/Home.jsx:97]: I think your prettify setting giving some extra spaces. You may want to check this one

https://discord.com/channels/982239924227031070/983221881723301899/1009049261826125844

File: public/index.html (hunk 1 above)
Comment (Sachin-chaurasiya) [reply public/index.html:40]: @atapas , This is the unique id created by umami.

Comment (Sachin-chaurasiya) [reply public/index.html:40]: > @atapas , This is the unique id created by umami and yes we should take it from .env

Comment (atapas) [reply public/index.html:40]: ok thanks, let's fix then.

**Review by atapas — APPROVED** [review]
